tests/commands/test_get_accounts.py

```diff
@@ -0,0 +1,120 @@
+"""Tests for get-accounts command."""
+import logging
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from nile.core.account import Account
+from nile.utils.get_accounts import _check_and_return_account, get_accounts
+
+NETWORK = "goerli"
+PUBKEYS = [
+    "883045738439352841478194533192765345509759306772397516907181243450667673002",
+    "661519931401775515888740911132355225260405929679788917190706536765421826262",
+]
+ADDRESSES = ["333", "444"]
+INDEXES = [0, 1]
+ALIASES = ["TEST_KEY", "TEST_KEY_2"]
+
+MOCK_ACCOUNTS = {
+    PUBKEYS[0]: {
+        "address": ADDRESSES[0],
+        "index": INDEXES[0],
+        "alias": ALIASES[0],
+    },
+    PUBKEYS[1]: {
+        "address": ADDRESSES[1],
+        "index": INDEXES[1],
+        "alias": ALIASES[1],
+    },
+}
+
+
+@pytest.fixture(autouse=True)
+def tmp_working_dir(monkeypatch, tmp_path):
+    monkeypatch.chdir(tmp_path)
+    return tmp_path
+
+
+@pytest.fixture(autouse=True)
+def mock_subprocess():
+    with patch("nile.core.compile.subprocess") as mock_subprocess:
+        yield mock_subprocess
+
+
+@pytest.mark.parametrize(
+    "private_keys, public_keys",
+    [
+        ([ALIASES[0], PUBKEYS[0]]),
+        ([ALIASES[1], PUBKEYS[1]]),
+    ],
+)
+def test__check_and_return_account_with_matching_keys(private_keys, public_keys):
+    # Check matching public/private keys
+    account = _check_and_return_account(private_keys, public_keys, NETWORK)
+
+    assert type(account) is Account
+
+
+@pytest.mark.parametrize(
+    "private_keys, public_keys",
+    [
+        ([ALIASES[0], PUBKEYS[1]]),
+        ([ALIASES[1], PUBKEYS[0]]),
+    ],
+)
+def test__check_and_return_account_with_mismatching_keys(private_keys, public_keys):
```

Comment:
I chose to double underscore because the func begins with an underscore (`_check_and_return_account`). I'll change it to a single, it looks a bit funky now

Reply:
nah, it's good